pkg/api/types.go

```diff
@@ -803,9 +803,10 @@ type Container struct {
 
 	// Variables for interactive containers, these have very specialized use-cases (e.g. debugging)
 	// and shouldn't be used for general purpose containers.
-	Stdin     bool `json:"stdin,omitempty"`
-	StdinOnce bool `json:"stdinOnce,omitempty"`
-	TTY       bool `json:"tty,omitempty"`
+	Stdin          bool `json:"stdin,omitempty"`
+	StdinOnce      bool `json:"stdinOnce,omitempty"`
+	TTY            bool `json:"tty,omitempty"`
+	ReadOnlyRootFS bool `json:"readOnlyRootFS,omitempty"`
```

Comment:
The comment for this section says these variables are for interactive containers only.  If this is not for debugging, it should be moved above this section with a comment explaining its purpose and if it is required.